lib/widgets/enable_backup_dialog.dart

```diff
@@ -196,59 +188,82 @@ class _BackupNowButtonState extends State<_BackupNowButton> {
 
     bool isRemoteServer = widget.backupSettings.backupProvider?.isRemoteServer;
 
-    return TextButton(
-      onPressed: (() async {
-        final currentProvider = widget.backupSettings.backupProvider;
-
-        // First we wish to attempt to backup before making the user manually
-        // having to select a backup provider.
-        if (widget.signInNeeded == false && currentProvider != null) {
-          Navigator.pop(context);
-          await ServiceInjector().breezBridge.requestBackup();
```

Comment:
Why do we remove this ?

Reply:
The library methods shouldn't be called directly from the UI.
We're missing crucial info on all courts if you compare this to _backupNow on BackupBloc.

The condition is the same, it did rely on BackupState errors and specifically BackupFailedException's caused by authentication errors before. However, since it was not read through stream it was not being updated. Which is a problem if our backup trigger attempt fails with the same reason. The UI was not aware of it until EnableBackupDialog is opened again.

Reply:
i understand, thank you for the clarification